File: android/src/main/java/com/phamhuu/native_image/option/Text.java
package com.phamhuu.native_image.option;

import android.graphics.Paint;

public class Text implements Option {
    public final String label;
    public final int size;
    public final Paint.Align textAlign;
    public final int gravity;
    public final String color;
    public final int horPadding;
    public final int verPadding;

    public Text(String label, int size, Paint.Align textAlign, int gravity, String color, int horPadding, int verPadding) {
        this.label = label;
        this.size = size;
        this.textAlign = textAlign;
        this.gravity = gravity;
        this.color = color;
        this.horPadding = horPadding;
        this.verPadding = verPadding;
    }

   static public Paint.Align setAlignText(int caseAlign) {
        switch (caseAlign) {
            case Text.CENTER:
                return  Paint.Align.CENTER;
            case Text.RIGHT:
                return  Paint.Align.RIGHT;
            default: return  Paint.Align.LEFT;
        }
    }
}

File: ios/Classes/option/Text.swift

import Foundation
class Text: Option{
    public init(label: String, sizeText: Int, textAlign: NSTextAlignment, garvity: Int, colorText: Int, horPadding: Int, verPadding: Int) {
        self.label = label
        self.sizeText = sizeText
        self.textAlign = textAlign
        self.garvity = garvity
        self.colorText = colorText
        self.horPadding = horPadding
        self.verPadding = verPadding
    }
    
    var label: String = ""
    var sizeText: Int
    var textAlign: NSTextAlignment
    var garvity: Int
    var colorText: Int
    var horPadding:Int
    var verPadding:Int
    
    static func setAlignText(align:Int)->NSTextAlignment{
        switch align {
        case center:
            return NSTextAlignment.center
        case right:
            return NSTextAlignment.right
        default :
        return NSTextAlignment.left
        }
    }
    
}
